Dockerfile

```diff
@@ -23,7 +31,7 @@ RUN ARCH="$(uname -m)" && \
 && BIN_LOCATION="$(tar -tzf /tmp/azcopy.tgz | grep "/azcopy")" \
 && export BIN_LOCATION \
 && tar -xvzf /tmp/azcopy.tgz --strip-components=1 --directory=/usr/bin/ "$BIN_LOCATION" \
-&& chown root:root /usr/bin/azcopy \
+&& chown "${user}:${user}" /usr/bin/azcopy \
```

Comment:
```suggestion
```

Binaries in `/usr/bin` should be owned by `root` (which is already the case once you've downloaded it as the default container user is... `root` until the instruction `USER` is set)